File: backend/src/main/java/br/com/csi/backend/model/Resposta.java
package br.com.csi.backend.model;

import javax.persistence.*;

import lombok.Data;

@Table(name = "Resposta")
@Entity(name="Resposta")
@Data
public class Resposta {

       @Id
       @GeneratedValue
       private Integer id_resposta;
       private String mensagem;
       private String status;
       private String titulo;
       @Column(name = "id_usuario")
       private Integer usuario;
       @Column(name = "id_solicitacao")
       private Integer solicitacao;
}

File: frontend/lib/model/Resposta.dart

class Resposta {

  int id_resposta;
  String mensagem;
  String status;
  String titulo;
  int usuario;
  int solicitacao;

  Resposta({
    required this.id_resposta,
    required this.mensagem,
    required this.status,
    required this.titulo,
    required this.usuario,
    required this.solicitacao
  });

  factory Resposta.fromJson(Map<String, dynamic> json){
    return Resposta(
        id_resposta: json['id_resposta'] as int,
        mensagem: json['mensagem'] as String,
        status: json['status'] as String,
        titulo: json['titulo'] as String,
        usuario: json['usuario'] as int,
        solicitacao: json['solicitacao'] as int,
    );
  }

  Map<String, dynamic> toJson() {
    return {
      'id_resposta': id_resposta,
      'mensagem': mensagem,
      'status': status,
      'titulo': titulo,
      'usuario': usuario,
      'solicitacao': solicitacao
    };
  }

}